Update mongodb.md (#1229)
percona/pmm-doc: docs/setting-up/client/mongodb.md

```

!!! note alert alert-primary ""

>     If you have already [added the MongoDB service to PMM](#add-service), you should restart the PMM agent service after changing the profiling level.

Reviewer: 👍🏼

>     If you have already [added the MongoDB service to PMM](#add-service), you should restart the PMM agent service after changing the profiling level.

Reviewer 2: ```suggestion
    If you have already [added the MongoDB service to PMM](#add-service), make sure to restart the PMM agent service after adjusting the profiling level.
```

## Add service

> When you have configured your database server, you can add a MongoDB service with the user interface or on the command line.

Reviewer 2: ```suggestion
"After configuring your database server, you can add a MongoDB service either through the user interface or via the command line.
```

When successful, PMM Client will print `MongoDB Service added` with the service's ID and name. Use the `--environment` and `-custom-labels` options to set tags for the service to help identify them.

!!! hint alert alert-success "Tips"

>     - When adding nodes of a sharded cluster, add each node separately using the `--cluster mycluster` option for the [MongoDB Cluster Summary](../../details/dashboards/dashboard-mongodb-cluster-summary.md) dashboard to populate correctly. 

Reviewer 2: ```suggestion
    - When adding nodes to a sharded cluster, ensure to add each node separately using the `--cluster mycluster` option. This allows the [MongoDB Cluster Summary](../../details/dashboards/dashboard-mongodb-cluster-summary.md) dashboard to populate correctly. 
```

>     - You can also use the `--replication-set` option to specify a replication set, altough they are automatically detected. Example: `--replication-set config` for your config servers; `--replication-set rs1` for your servers in the first replica set, `--replication-set rs2` for your servers in the second replica set, and so on.

Reviewer: > altough they are automatically detected

Are they? Most of dashboards keep working, because we just use another label. But for backups as far as I remember it's critical to pass that flag.

Author: well, we can derive the replication set name from metrics, so it should be auto detected. Otherwise the user could supply the wrong replication-set which would cause all sorts of problems.

Reviewer: you are right, but it's not about can we or can't we. it's about do we or don't we. --replication-set generates label that is different from `set` label, so until we formalise and make autodetect I wouldn't mention that in documentation.

Author: I agree. To validate if it's really needed, I ran a quick test and backups seems to work fine even without providing the --replication-set flag. Do you have any example or remember under which conditions may the backup fail?

Reviewer: No, that was just an assumption, I checked a code and seems like we use RS for PITR, but it's read from Mongo itself.

>     - You can also use the `--replication-set` option to specify a replication set, altough they are automatically detected. Example: `--replication-set config` for your config servers; `--replication-set rs1` for your servers in the first replica set, `--replication-set rs2` for your servers in the second replica set, and so on.

Reviewer 2: ```suggestion
    - You can also use the `--replication-set` option to specify a replication set, altough they are automatically detected. For instance, you can use `--replication-set config` for your config servers; `--replication-set rs1` for your servers in the first replica set, `--replication-set rs2` for your servers in the second replica set, and so on.
```

>     - Atlas doesn't support direct connections. When connecting to an Atlas instance, use the `pmm-admin` option `--direct-connection=false`. (Doing so will prevent replicaset status from working and the MongoDB Overview dashboard widget will show invalid values.)

Reviewer 2: ```suggestion
    - When connecting to an Atlas instance, keep in mind that direct connections are not supported. Make sure to use the `--direct-connection=false` option with `pmm-admin`. Otherwise, replicaset status functionality will not work as expected, and the MongoDB Overview dashboard widget will show invalid values.
```
